joemam

```diff
@@ -0,0 +1,2 @@
+<h1> hi there </h1> 
```

Comment:
Missing capital letters. "hi" should be "Hi". Optionally, "there" could be changed to "There" or one of !/./? should be included at the end.